src/components/DownloadEmpty.js

```diff
@@ -3,7 +3,7 @@ import { Box, Heading } from 'grommet'
 import { useResponsive } from 'hooks/useResponsive'
 import { Button } from 'components/Button'
 
-export const EmptyDataset = () => {
+export const DownloadEmpty = () => {
```

Comment:
I've renamed and prefixed this component with `Download` as this is used for the empty state of the download page.